src/components/anonymous/ChoosePassword.js

```diff
@@ -198,14 +202,15 @@ function ChoosePassword({ match }) {
                 <div className="rf-col-offset-md-3" ></div>
 
                 <div className="rf-col-offset-md-3" ></div>
-                <div className="rf-col-12 rf-col-md-6 rf-mb-8w rf-mb-md-15w recapitulatif">Votre boite mail @conseiller-numerique.fr sert d’identifiant<br />
+                <div className="rf-col-12 rf-col-md-6 rf-mb-8w rf-mb-md-15w recapitulatif">Votre boite mail <strong>{slugify(`${user?.prenom} ${user?.nom}`, { replacement: '.', lower: true, strict: true })}@conseiller-numerique.fr</strong> sert d’identifiant<br />
```

Comment:
il y a un petit warning de l'eslint : `This line has a length of 264. Maximum allowed is 160`